hackathon_site/dashboard/frontend/src/components/teamDetail/TeamPendingOrderTable/TeamPendingOrderTable.tsx

```diff
@@ -11,7 +11,7 @@ import {
     TableHead,
     TableRow,
 } from "@material-ui/core";
-import { mockHardware, mockPendingOrdersInTable } from "testing/mockData";
+import { OrderInTable, OrderStatus, Hardware } from "api/types";
```

Comment:
I think you should add this updateThunk to SimplePendingOrderFulfillmentTable instead (since for the TeamPendingOrderTable the fulfillment will be more complicated than just a status change)